migrate/playbooks/rds-timescale-live-migration.md

```diff
@@ -0,0 +1,410 @@
+
+8. Once you have verified the new parameter group value, select "Apply immediately" and click
+on "Modify DB Instance".
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-live-migration/create-pg-10.jpg"
+ alt="Apply DB parameter group"/>
+<Highlight type="important">
+Configuring the new parameter group will not cause your database to be restarted. In the next
+step you will manually reboot the database.
+</Highlight>
+
+9. Your RDS service will be in the "Modifying" state for a short while, after which, you should
+see that the parameter group is in the "Pending reboot" state and must be rebooted to be applied.
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-live-migration/create-pg-11.jpg"
+ alt=""/>
+
+10. Manually reboot your service in order to apply the changes. In the "Actions" dropdown, click
+"Reboot" and then "Confirm".
+
+11. After your service reboots, ensure that the new "DB parameter group" is applied in your RDS
+instance.
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-live-migration/create-pg-12.jpeg"
+ alt="Ensure the new DB parameter group on your RDS instance"/>
+
+12. Reconnect to your RDS instance and verify the configuration parameter values.
+```sh
+postgres=> show wal_level;
+ wal_level
+-----------
+ logical
+(1 row)
+
+postgres=> show old_snapshot_threshold ;
+ old_snapshot_threshold
+------------------------
+ -1
+(1 row)
+```
+
+### Prepare an intermediate machine
+Live Migration must be executed from a dedicated machine. The live migration tools
+connect to the source and target databases, and all data will flow through the
+dedicated machine between the source and the target. This instance must be able
+to access your RDS service and ideally is in the same region as your Timescale
+instance. We will set up an EC2 instance in the same VPC as the RDS service.
+
+#### Create an EC2 instance
+1. In the AWS search, type "EC2" and select the "EC2" option under services.
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-live-migration/search-ec2.jpg"
+ alt="Search for EC2 services"/>
+
+2. Click on "Launch instance".
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-live-migration/launch-ec2-instance.jpg"
+ alt="Launch EC2 instance"/>
+
+3. Configure your EC2 instance.
+* For "Instance type", use 2 CPU and 8 GB memory at least. If your migration
+involves a larger database, you should choose accordingly.
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-live-migration/config-instance-type.jpg"
+ alt="Configure instance type"/>
+
+<Highlight type="info">
+The procedures outlined in this, as well as subsequent sections, are based on the
```

Comment:
Instead of confusing the user by letting them pick the OS in the above step and mentioning that the below steps are targeted for Ubuntu, I would suggest removing this point and instead mentioning earlier to pick Ubuntu as the intermediate image.

Reply:
I think this should be fine. We just mention that we wrote steps based on ubuntu image. If a user wants some other imagine (due to some regulation/preference), he is free to do so.

Reply:
Done. I have added `a. For "Application and OS image", choose Ubuntu Server LTS.` above this point.